packages/app-mobile/components/screens/ConfigScreen/plugins/PluginStates.tsx

```diff
@@ -126,6 +127,11 @@ const PluginStates: React.FC<Props> = props => {
 
 	return (
 		<View>
+			<SectionDescription
+				title={`${_('Plugins')} (Beta)`}
```

Comment:
As is done elsewhere in the code, `(Beta)` is marked as untranslatable.

An alternative could be to do something similar to the following:
```suggestion
				title={_('%s (Beta)', _('Plugins'))}
```

Reply:
This is fine as it is, because normally beta feature don't remain in this state so we don't want to do anything special in terms of translation.